packages/dds/tree/src/schema-stored/Stored and View Schema.md

```diff
@@ -35,104 +34,116 @@ Places we can store stored schema information:
 
 # Example/Proposed Design
 
 This design shows an easy to implement MVP, and is not intended to have document or API compatibility with what we do longer term.
-It minimally implements the requirements from "What options to support" while being extensible in the future.
-It can be subsetted for use in earlier milestones, and could be extended for later ones, see "Schedule" below.
+
+It minimally implements the requirements from [Options to Support](#options-to-support) while being extensible in the future.
+It can be subsetted for use in earlier milestones, and could be extended for later ones, see [Schedule](#schedule).
 
 This design is intentionally minimal while still being useful.
 Features not needed for minimal strongly typed application use or the JSON and XML domain schema are not included.
 
 ## Schema Representation
 
-The MVP should include a simple schema system, usable as a MVP for both `stored schema` and `view schema`.
+The MVP should include a simple schema system, usable as an MVP for both `stored schema` and `view schema`.
 
 One possible such schema system is included in [Schema.ts](./Schema.ts).
 
 These `TreeSchema` and `FieldSchema` can be added as `stored schema` as part of an edit op,
-which is considered conflicted if it tries to add a type that has a conflicting `name` and is not equal to the existing one.
+which is considered conflicted if it tries to add a type that has a conflicting `name` and is not equal to the existing one. TODO: This was confusing to me. This might be a bit more clear as part of the "Use as `Stored Schema` section.
 
-While a schema must be added to the document as a stored schema to use the type
+While a schema must be added to the document as a stored schema to use the type TODO: Which type? I.e. does this mean the same thing as 'a schema must be added to the document as a stored schema in order to be applied to a specific type of node'?
 (otherwise adding a new type could break existing data which might not even be downloaded on the current client),
 it's possible to add a schema that is compatible with all possible data (assuming the children themselves are compatible with their own types).
+
 Applications which wish to rely entirely on schema-on-read for some or all of their data can use this pattern for all `stored schema` and only use their actual developer authored schema as `view schema`:
 
 ## Use as `Stored Schema`
 
--   Document load: The application can check that their view schema match stored schema, and check that the root has supported content.
+TODO: Use what as stored schema? Isn't stored schema just that or can it be used as something else? Or is 'Usages of Stored Schema` a more accurate title?
+
+-   Document load: The application can check that their view schema matches their stored schema and that the root has supported content.
+
     See `checkCompatibility` in [Schema.ts](./Schema.ts) for an example of how this could work.
```

Comment:
this link is wrong now. Also this whole bullet point is probably better covered below talking about view schema.